app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,49 @@
+<nav>
+ <div class="container">
+   <% if user_signed_in? %>
+    <%= link_to "Fakebook", authenticated_root_path, id: "logo", class: "logo-authenticated" %>
+   <% else %>
+    <%= link_to "Fakebook", unauthenticated_root_path, id: "logo" %>
+   <% end %>
+
+   <ul class="nav navbar-nav navbar-right">
+     <% if user_signed_in? %>
+       <div class="navbar-right">
+         <div class="navbar-right-links">
+           <%= link_to('Find Friends', find_friends_path) %>
+           <%= link_to('<i class="material-icons">notifications</i>'.html_safe, notifications_path) %>
```

Comment:
Instead of using html tags inside string + html_safe, you can just use erb functions:
```<a href=<%= notifications_path %> >
            <i class="material-icons">notifications</i>
           </a>
```

Reply:
Fixed :-)